contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.7.0--2.8.0.sql

```diff
@@ -0,0 +1,15 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
```

Comment:
Can you check https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.6.0--2.7.0.sql to find what other changes we need here? For example, we may need the following changes:
```
CALL sys.babelfish_update_user_catalog_for_guest_schema();
```
```
CREATE OR REPLACE PROCEDURE babelfish_drop_deprecated_object(object_type varchar, schema_name varchar, object_name varchar) AS
$$
```
```
-- After upgrade, always run analyze for all babelfish catalogs.
CALL sys.analyze_babelfish_catalogs();
```